mapproxy/cache/tile.py

```diff
@@ -162,10 +170,19 @@ def _load_tile_coords(self, tiles, dimensions=None, with_metadata=False,
         self.cache.load_tiles(tiles, with_metadata)
 
         for tile in tiles:
-            if tile.coord is not None and not self.is_cached(tile, dimensions=dimensions):
-                # missing or staled
-                uncached_tiles.append(tile)
-
+            if tile.coord is not None:
+                if not self.is_cached(tile, dimensions=dimensions):
+                    # missing or staled
+                    uncached_tiles.append(tile)
+                    tile.cache_hit = False if self.show_cache_status == True else None
+                    self.cache_misses = self.cache_misses + 1
+                else:
+                    tile.cache_hit = True if self.show_cache_status == True else None
+                    self.cache_hits = self.cache_hits + 1
+        cache_count = self.cache_hits + self.cache_misses
+        if (cache_count % 1000 == 0):
+            log.info("Tile caching statistics: HITS: %d (%0.1f%)\t MISSES: %d (%0.1f%)" %
```

Comment:
Seems that this is logged even with show_cache_status=False.
Not sure if this is intended, because this may "spam" the logs and cannot be disabled